ols/src/query_helpers/docs_summarizer.py

```diff
@@ -77,7 +82,20 @@ def _get_rag_data(self, rag_index: VectorStoreIndex, query: str) -> list[dict]:
         )
         # Truncate rag context, if required.
         token_handler_obj = TokenHandler()
-        interim_prompt = CHAT_PROMPT.format(context="", query=query, chat_history=[])
+
+        rag_context = ""
+        prompt, _ = generate_prompt(
+            self.provider,
+            self.model,
+            model_options,
+            query,
+            [],
+            rag_context,
+        )
```

Comment:
isn't this going to generate a prompt that doesn't have the rag context stanza, since you're passing an empty string as the rag context?

Reply:
it should be non-empty string, typo, fixed